### DISCUSSION TITLE
Where are embedding models downloaded to?

### DISCUSSION THREAD

**Discussion by jonmach** [huggingface/text-embeddings-inference#147] [Q&A] [answered]
Starting a server with a supported model works well, and with the docker examples in the documentation, it's clear where the data will be downloaded to: e.g. "-v $volume:/data".

However, I built a local install (M2 with Metal) and I can't see where the models are being saved to.  From the code (download.rs), it isn't clear where it's saving, and then pulling the model from

Is it also worth considering whether a server could start with a parameterized data directory?

**Comment by OlivierDehaene** [comment, marked as answer]
The model are downloaded by default to `~/.cache/huggingface`. If you want to change the default directory, you can use the `HUGGINGFACE_HUB_CACHE` env var or `--huggingface-hub-cache` arg.
